### PR TITLE
DropCommunicatorUtil is now a Module

### LINKED ISSUE #319: Fix liveCycleTest
```liveCycleTest``` randomly fails in Travis. This false positive has to be fixed.

Edit: Further the name probably should be li**f**eCycleTest.

### PR DESCRIPTION
**Pull request by Gottox**
also fixes #319

### CODE REVIEW

File: src/main/java/de/qabel/core/module/ModuleManager.java [hunk 1]
```diff
@@ -87,8 +98,8 @@ public void run() {
 	 * @throws InstantiationException
 	 * @throws IllegalAccessException
```
Comment (L-Henke) [inline src/main/java/de/qabel/core/module/ModuleManager.java]: The newly thrown exceptions are missing in the Javadoc.

File: src/main/java/de/qabel/core/module/ModuleManager.java [hunk 2]
```diff
@@ -87,8 +98,8 @@ public void run() {
...
 	 */
-	public void startModule(Class<?> module) throws InstantiationException, IllegalAccessException {
-		Module m = (Module) module.newInstance();
+	public void startModule(Class<?> module) throws InstantiationException, IllegalAccessException, NoSuchMethodException, InvocationTargetException {
+		Module m = (Module) module.getConstructor(EventEmitter.class).newInstance(eventEmitter);
```
Comment (jan-schreib) [inline src/main/java/de/qabel/core/module/ModuleManager.java]: EventEmitter does not have a constructor, so this can't work.

**Comment by thechauffeur** [comment]
@L-Henke because you kind of take care of the ackack relating work now please have a look at this PR. Any suggestions are helpful.

**Comment by L-Henke** [comment]
Will be redone. See #338

**Pull request closed without merging**